### PR TITLE
Add LeakyRelu and Sigmoid QLinear Quantization support

### PR DESCRIPTION
**Pull request by zhanghuanrong**

### CODE REVIEW

File: onnxruntime/python/tools/quantization/operators/activation.py [hunk 1]
```diff
@@ -20,3 +21,49 @@ def quantize(self):
...
+            return
+
+        # No assert on op_type as it is controlled by registry
+        # only try to quantize when given quantization parameters for it
+        data_found, output_scale_name, output_zp_name, _, _ = self.quantizer._get_quantization_params(node.output[0])
+        if (not data_found):
+            super().quantize()
+            return
+
+        (quantized_input_names, zero_point_names, scale_names, nodes) = self.quantizer._quantize_inputs(node, [0])
```
Comment (RandySheriffH) [inline onnxruntime/python/tools/quantization/operators/activation.py:38]: >( [](start = 8, length = 1)

nit: do we need parenthesis here?

File: onnxruntime/python/tools/quantization/operators/activation.py [hunk 2]
```diff
@@ -20,3 +21,49 @@ def quantize(self):
...
+
+        qlinear_activation_output = node.output[0] + "_quantized"
+        qlinear_activation_name = ""
+        if node.name != "":
+            qlinear_activation_name = node.name + "_quant"
+        kwargs = {}
+        for attribute in node.attribute:
+            kwargs.update(_attribute_to_kwarg(attribute))
+        kwargs["domain"]=ms_domain
```
Comment (RandySheriffH) [inline onnxruntime/python/tools/quantization/operators/activation.py:47]: >]=m [](start = 23, length = 3)

nit: spacing.

File: onnxruntime/python/tools/quantization/operators/activation.py [hunk 3]
```diff
@@ -20,3 +21,49 @@ def quantize(self):
 
         quantized_value = self.quantizer.quantized_value_map[node.input[0]]
         self.quantizer.quantized_value_map[node.output[0]] = quantized_value
+
+    def quantize(self):
+        node = self.node
+        if (node.op_type == "Relu" or node.op_type == 'Clip'):
```
Comment (RandySheriffH) [inline onnxruntime/python/tools/quantization/operators/activation.py:27]: >( [](start = 11, length = 1)

nit: guess '==' has higher precedence already.
[https://www.mathcs.emory.edu/~valerie/courses/fall10/155/resources/op_precedence.html](https://www.mathcs.emory.edu/~valerie/courses/fall10/155/resources/op_precedence.html)

File: onnxruntime/python/tools/quantization/operators/activation.py [hunk 4]
```diff
@@ -20,3 +21,49 @@ def quantize(self):
...
+
+        (quantized_input_names, zero_point_names, scale_names, nodes) = self.quantizer._quantize_inputs(node, [0])
+
+        qlinear_activation_output = node.output[0] + "_quantized"
+        qlinear_activation_name = ""
+        if node.name != "":
+            qlinear_activation_name = node.name + "_quant"
+        kwargs = {}
+        for attribute in node.attribute:
+            kwargs.update(_attribute_to_kwarg(attribute))
```
Comment (RandySheriffH) [inline onnxruntime/python/tools/quantization/operators/activation.py:46]: nit: _ = [kwargs.update(...) for attr in node.attribute] ?

File: onnxruntime/python/tools/quantization/operators/activation.py [hunk 5]
```diff
@@ -20,3 +21,49 @@ def quantize(self):
...
+
+        qlinear_activation_inputs = []
+        # Input 0
+        qlinear_activation_inputs.append(quantized_input_names[0])
+        qlinear_activation_inputs.append(scale_names[0])
+        qlinear_activation_inputs.append(zero_point_names[0])
+
+        # Output
+        qlinear_activation_inputs.append(output_scale_name)
+        qlinear_activation_inputs.append(output_zp_name)
```
Comment (RandySheriffH) [inline onnxruntime/python/tools/quantization/operators/activation.py:57]: nit: why not just assign like:

qliner... = [a,b,c,d,e]

File: onnxruntime/python/tools/quantization/operators/activation.py (hunk 2 above)
Comment (zhanghuanrong) [reply onnxruntime/python/tools/quantization/operators/activation.py]: fixed.

File: onnxruntime/python/tools/quantization/operators/activation.py (hunk 3 above)
Comment (zhanghuanrong) [reply onnxruntime/python/tools/quantization/operators/activation.py:27]: () is ok.

File: onnxruntime/python/tools/quantization/operators/activation.py (hunk 5 above)
Comment (zhanghuanrong) [reply onnxruntime/python/tools/quantization/operators/activation.py]: Maybe better. changed.

File: onnxruntime/python/tools/quantization/operators/activation.py (hunk 3 above)
Comment (zhanghuanrong) [reply onnxruntime/python/tools/quantization/operators/activation.py:27]: habit.

**Review by yufenglee — APPROVED** [review]
:shipit:

**Review by yufenglee — APPROVED** [review]
:shipit:

**Pull request merged**